### PR TITLE
Welcome Bolddesk by Syncfusion, a new KEDA end-user!

### PR DESCRIPTION
**Pull request by kailashp2289**
Added new user called BoldDesk by Syncfusion.

Signed-off-by: Kailash P <kailashp@syncfusion.com>

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Comment by kailashp2289** [comment]
Signed-off-by: Kailash P <kailashp@syncfusion.com>

File: config.toml [hunk 1]
```diff
@@ -445,6 +445,10 @@ logo = "teamleader.png"
 url = "https://zapier.com"
 logo = "zapier.png"
 
+[[params.users]]
+url = "https://www.bolddesk.com"
+logo = "bolddesk.svg"
```
Comment (tomkerkhove) [inline config.toml:450]: Sorry for the nit, but can you alphabetically sort it please?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: config.toml (hunk 1 above)
Comment (kailashp2289) [reply config.toml]: i have reordered it alphabetically, can you review now?

**Review by tomkerkhove — APPROVED** [review]

**Comment by tomkerkhove** [comment]
Can you fix the DCO issues please? You can learn more in our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what).

**Pull request closed without merging**